### PR TITLE
Add Styling According to Zeplin Design

### PR DESCRIPTION
**Pull request by ahmidou09**
This pull request implements the required styling for the Bookstore project based on the Zeplin design.
All elements have been styled according to the provided guidelines.

**Changes Made**
     - Implemented styling for all components as per Zeplin design.
     - Ensured consistency with CSS rules mentioned in Zeplin.
     - Verified responsiveness and cross-browser compatibility.

### CODE REVIEW

File: src/components/Home/IndividualBook.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  opacity: 0.6;
+  font-size: 0.84rem;
+  font-weight: bold;
+}
+
+.btn {
+  background-color: transparent;
+  cursor: pointer;
+  border: none;
+}
```
Comment (Johnadibe) [inline src/components/Home/IndividualBook.css]: > Well done, @ahmidou09!👍 Consider incorporating the design's identical color to enhance the visual consistency. This adjustment will further enhance your work.😊
> 
> Design:
> 
> action2-1
> 
> Your design:
> 
> action2
>

File: src/components/Home/IndividualBook.js [hunk 2]
```diff
@@ -23,16 +28,44 @@ function IndividualBook({ id, book }) {
...
   return (
     <div className="book">
-      <header className="book-header">
-        <span className="book-category">{category}</span>
-        <h2 className="book-title">{title}</h2>
-        <h3 className="book-author">{author}</h3>
-        <div className="book-actions">
-          <button type="button" className="btn btn-delete" onClick={handleDelete}>
+      <div className="detail">
+        <span className="category">{category}</span>
```
Comment (Johnadibe) [inline src/components/Home/IndividualBook.js:32]: > - [ ] I have noticed that when I initially add a book, the pre-selected default category (Action) JSX doesn't appear on the page. My recommendation is to disable the default as a clickable option, allowing the Action category to be displayed subsequently. 👍
> 
> See Design below:
> 
> action
> 
> Your design:
> 
> action1
> 
> See the video below for more clearification:
> 
> [screen-capture.webm](https://github.com/ahmidou09/Redux-Bookstore/assets/40009354/6b806116-945b-41fa-aaca-45dd78a2748c)

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @ahmidou09👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used Git Flow correctly 👍🏽
✅ Linter checks passed✔️
✅ Used the CSS rules detailed in Zeplin to style the website✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Mehdi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job making the requested changes.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**